src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --h: 60px;
+  --c: #00acdc;
```

Comment:
Consider using meaningful names for variables, for example '--accent-color', etc.